dockerfiles/run/guice/memory/Dockerfile

```diff
@@ -2,7 +2,7 @@
 #
 # VERSION	1.0
 
-FROM openjdk:8u222-jre
+FROM adoptopenjdk:11-jre-hotspot
```

Comment:
you don't fix the version?

Reply:
Good question.
On one hand, build is less stable because the jdk can change between two builds.
One the hand, building an image means you have the last security updates.
In this particular case, given the stability of the JDK and the rate of our changes to base images, I would choose automatic update by not being specific on the jdk version.